cli.js

```diff
@@ -11,15 +12,53 @@ function _pickConfig (config) {
   /* eslint-enable */
 }
 
+const parser = argly
+  .createParser({
+    '--help -h': {
+      type: 'string',
+      description: 'Show this help message'
+    },
+    '--version -v': {
+      type: 'string',
+      description: 'Show the version number of mocha-puppeteer'
+    },
+    '--pattern -p *': {
+      type: 'string',
+      description: 'Pattern to run tests. Either a single file or glob pattern.'
+    }
+  })
+  .example('Test a single file: "mocha-puppeteer /foo/bar-test.js"')
+  .example('Test a series of files using a glob pattern: "mocha-puppeteer /foo/**/*-test.js"')
```

Comment:
Can you change this to `/foo/*/*-test.js`? The double asterisk (`**`) only works certain shells (zsh and maybe others, but not bash)